src/tree/AttachedAccountsTreeItem.ts

```diff
@@ -107,12 +111,24 @@ export class AttachedAccountsTreeItem extends AzureParentTreeItem {
 
         const attachedAccounts: AzureTreeItem[] = await this.getAttachedAccounts();
 
-        return attachedAccounts.length > 0 ? attachedAccounts : [new GenericTreeItem(this, {
-            contextValue: 'cosmosDBAttachDatabaseAccount',
-            label: 'Attach Database Account...',
-            commandId: 'cosmosDB.attachDatabaseAccount',
-            includeInTreeItemPicker: true
-        })];
```

Comment:
Please verify that adding this new tree item isn't breaking the detach logic, which checks for a specific tree item when determining if its possible to detach.
https://github.com/microsoft/vscode-cosmosdb/blob/8d86a075ef090aec387c9a9c3861b42f1c383e8b/src/extension.ts#L105-L123

Reply:
I just checked, detach is working for both scenarios.